code/game/machinery/iv_drip.dm

```diff
@@ -347,6 +350,10 @@
 /obj/machinery/iv_drip/plumbing/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/plumbing/iv_drip, anchored)
+	AddComponent(/datum/component/simple_rotation)
+
+/obj/machinery/iv_drip/plumbing/set_transfer_rate_max()
+	return //alt clicking this shouldn't do anything special as it's reserved for rotation
```

Comment:
```suggestion
/obj/machinery/iv_drip/plumbing/AltClick(mob/user)
	return // This hotkey is BLACKLISTED since it's used by /datum/component/simple_rotation
```

Reply:
Try this instead, although it needs to be tested.

Reply:
doesn't work since the send signal is on /atom/AltClick it still needs to call parent in the altclick proc

Reply:
If the default transfer rate is already max, do we really need a hotkey to set the transfer rate to max?  For atmos pipes I could understand, but for IV drips?  I don't play with chemistry or plumbing so I don't know how common of a thing this is.

Reply:
I might as well just remove it entirely yeah